config/settings/base.py

```diff
@@ -456,3 +455,16 @@
 
 with open(os.path.join(BASE_DIR, "package.json"), encoding="utf-8") as fd:
     ADSERVER_VERSION = json.load(fd)["version"]
+
+# Additional middleware for setting the user's real IP
+# and translating the IP into a geo for ad targeting.
+ADSERVER_IPADDRESS_MIDDLEWARE = env("ADSERVER_IPADDRESS_MIDDLEWARE", default=None)
+ADSERVER_GEOIP_MIDDLEWARE = env("ADSERVER_GEOIP_MIDDLEWARE", default=None)
```

Comment:
Need to remember to set these on deploy.

Reply:
I think the deploy this goes out in will be this and nothing else =)

Reply:
I wonder if we should set the defaults to the base classes `GeoIpMiddleware` and `IpAddressMiddleware`. That way we could guarantee that `request.ip_address` and `request.geo` will always be present.

Reply:
I made these the default.